sentry/src/main/java/io/sentry/TraceContext.java

```diff
@@ -389,4 +316,34 @@ private Exception missingRequiredFieldException(String field, ILogger logger) {
       return exception;
     }
   }
+
+  @Override
```

Comment:
If this is needed for testing, I'd rather check the equality field per field, adding such methods increase the bundle size, and its not needed at runtime.